elements/urth-core-dataframe/urth-core-query-agg.html

```diff
@@ -0,0 +1,83 @@
+!--
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+This element is used to define an aggregate function to performend on grouped data. It is intended
```

Comment:
`performend` = `perform`
`...function in for the type...` = `...function for the type...`